schemas/parent.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: parent
+  title: Parent
+  group: 2
+  short: These fields contain information about a parent.
+  description: >
+    These fields contain information about a parent.
+
+    These fields provide more context about the parent process
+    and thread that are related to the data in the document.
+  type: group
+  fields:
+    - name: process
+      level: extended
+      type: object
+      description: >
+        Process.
+
+    - name: thread
```

Comment:
I've never seen anything or any use cases for referring to a "parent thread". I think we should be fine sticking with our current process.parent paradigm